TechnicoServer/TechnicoDomain/Services/ΙPropertyItemService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
```

Comment:
I notice this file is not used and is commented. If you don't use it, you should better remove it from your repository.